api/v1alpha1/clusterqueue_types.go

```diff
@@ -134,14 +134,19 @@ type ClusterQueueSpec struct {
 	Cohort string `json:"cohort,omitempty"`
 
 	// QueueingStrategy indicates the queueing strategy of the workloads
-	// across the queues in this Capacity. This field is immutable.
+	// across the queues in this ClusterQueue. This field is immutable.
 	// Current Supported Strategies:
 	//
 	// - StrictFIFO: workloads are sorted strictly by creation time.
 	//
 	// +kubebuilder:default=StrictFIFO
 	// +kubebuilder:validation:Enum=StrictFIFO
 	QueueingStrategy QueueingStrategy `json:"queueingStrategy,omitempty"`
+
+	// namespaceSelector defines which namespaces are allowed to submit workloads to
```

Comment:
What's the behavior if `nil`?

Reply:
selects nothing, which is the default.

Reply:
So if it's not set, then all the workloads are rejected? How come our integration tests didn't need to be updated to include a selector?

Reply:
It is set to an empty selector in the wrapper, which selects everything.

Reply:
I updated the comment to clarify that empty selector is allowed.